js/ui/date_box/ui.date_view_roller.js

```diff
@@ -216,35 +228,48 @@ const DateViewRoller = Scrollable.inherit({
         return Math.max(Math.min(index, itemsCount - 1), 0);
     },
 
+    _isInNullVicinity: function(x) {
```

Comment:
```suggestion
    _isInCurrentItemArea: function(delta) {
```
// `neighbourhood` seems a more reliable term in this context

Reply:
fixed